composer.json

```diff
@@ -24,13 +24,14 @@
     "require": {
         "php": "^7.1",
         "sonata-project/classification-bundle": "^3.6",
-        "sonata-project/core-bundle": "^3.13.7",
         "sonata-project/datagrid-bundle": "^2.3",
         "sonata-project/doctrine-extensions": "^1.5.1",
         "sonata-project/easy-extends-bundle": "^2.5",
+        "sonata-project/form-extensions": "^0.1 || ^1.4",
         "sonata-project/formatter-bundle": "^3.4 || ^4.0",
         "sonata-project/intl-bundle": "^2.4",
         "sonata-project/media-bundle": "^3.10",
+        "sonata-project/twig-extensions": "^0.1 || ^1.3",
```

Comment:
Do we need twig extensions here?

Reply:
https://github.com/sonata-project/SonataNewsBundle/blob/3.x/src/Status/CommentStatusRenderer.php#L17-L22